### ISSUE TITLE
:rename is stripped

### ISSUE THREAD

**Issue by vemv** [gfredericks/how-to-ns#5]
Hi!

The following clause:

```
[h.d.p :as d.p :refer [some-really-longass-name] :rename {some-really-longass-name short}])
```

Is transformed into

```
[h.d.p :as d.p :refer [some-really-longass-name]])
```

which breaks programs.

_(I use longer names than h.d.p - that's just redacted data)_

I realise that the styleguide forbids :rename, but still the library should err on the cautious side, shouldn't it?

Cheers - Victor

**Comment by gfredericks** [comment]
I agree! I had forgotten that the style guide forbids it, which makes me inclined to make it an option you can enable, similar to docstring requirements.

I'd be happy to see a PR, otherwise I will be able to work on this in a few days.

**Comment by vemv** [comment]
Thanks so much for this one!

One day I'll contribute to the project, have a couple ideas floating in my head :)

**Comment by gfredericks** [comment]
I added an `:allow-rename?` option, released as `0.1.8`; let me know if you encounter any issues.
